README.md

```diff
@@ -1,2 +1,66 @@
-# wodin
-odin on the web
+# WODIN
+
+[odin](https://github.com/mrc-ide/odin) on the web
+
+## Introduction
+
+WODIN is a web application with [Node.js](https://nodejs.org/en/) back end using [Express](http://expressjs.com/), 
+and [Vue.js](https://vuejs.org/) front end. Both parts are written in [Typescript](https://www.typescriptlang.org/).  
+
+The application is intended to be deployable with custom configuration allowing (typically) teaching courses to be delivered
+where students can author and run model code using [odin](https://github.com/mrc-ide/odin). 
+Odin will be invoked by WODIN to provide executable javascript
+models given odin code written by the users or provided in config. 
+
+WODIN configuration defines multiple 'apps', corresponding to teaching modules, which each provide functionality for 
+one of three types of model usage:
+- *basic* - to teach the basics of odin model definitions
+- *fit* - to fit a model to data
+- *stochastic* - to run a model stochastically
+
+Each app will have one of these three app types, as well as any further configuration required e.g. default odin code. 
+
+## Usage and configuration
+
+You can install dependencies, build and run the app locally in one step using `./scripts/build-and-run.sh`. The app will be available at http://localhost:3000 
+You may need to install typescript: `npm install -g typescript`
+
+WODIN will be deployable as a docker image which can be mounted with custom configuration as follows:
+
+### *.config.json
+
+Each config.json file defines config values for an app. The app will be available at `/apps/[appName]` for a config
```

Comment:
Not 100% sure about this specific, but I think that some prefix between the root and the page is probably sensible (especially if we ever needed any sort of config/admin/metrics/login page).

So when deployed we'll have `https://wodin.dide.ic.ac.uk/apps/ebola` or `https://www.dide.ic.ac.uk/wodin/apps/ebola` depending on how we might configure this?

If we have the ability to change `apps` later without too much pain this is fine for now

Reply:
The `apps` path could be configurable per instance if that would be useful.